### PR TITLE
Implement restore default on customize dialog

### PR DESCRIPTION
**Pull request by ItsMurumba**

### CODE REVIEW

File: packages/transaction-log/src/components/common/transactionlog.datatable.component.tsx [hunk 1]
```diff
@@ -92,63 +92,101 @@ const TransactionLogTable: React.FC<{
   }
 
   return (
-    <Box sx={{padding: '16px'}}>
+    <Box>
```
Comment (brett-onions) [inline packages/transaction-log/src/components/common/transactionlog.datatable.component.tsx:95]: could this <Box> be removed?

**Review by brett-onions — CHANGES_REQUESTED** [review]

File: packages/transaction-log/src/interfaces/index.interface.ts [hunk 2]
```diff
@@ -65,10 +65,26 @@ export interface CustomizeDialogProps {
...
+    path: boolean
+    param: boolean
+    client: boolean
+    method: boolean
+  }
   handleFilterVisibilityChange: (
     event: React.ChangeEvent<HTMLInputElement>
   ) => void
+  onRestoreDefaults: () => void
+  isDefaultState: boolean
```
Comment (marvinIsSacul) [inline packages/transaction-log/src/interfaces/index.interface.ts:87]:
```suggestion
  isDefaultState?: boolean
```

**Review by marvinIsSacul — APPROVED** [review]

File: packages/transaction-log/src/components/common/transactionlog.datatable.component.tsx (hunk 1 above)
Comment (ItsMurumba) [reply packages/transaction-log/src/components/common/transactionlog.datatable.component.tsx:95]: this is already removed @brett-onions

File: packages/transaction-log/src/interfaces/index.interface.ts (hunk 2 above)
Comment (ItsMurumba) [reply packages/transaction-log/src/interfaces/index.interface.ts:87]: Why would you want to make the property optional? @marvinIsSacul

**Review by brett-onions — APPROVED** [review]

**Pull request merged**